src/stats/stats.jl

```diff
@@ -58,6 +61,12 @@ function show(io :: IO, stats :: ExecutionStats)
   println("  feasibility measure: $(stats.feas_norm)")
   println("  iterations: $(stats.iter)")
   println("  elapsed time: $(stats.elapsed_time)")
+  if length(stats.solver_specific) > 0
+    println("  solver specifics:")
+    for (k,v) in stats.solver_specific
+      println("    $k: $v")
```

Comment:
With `io` do we need to use `show`? I'm trying to make something useful for `show` and nicer for `print`, but it's not looking good. Maybe we can add a function to return the json if the user wants more printing control. What use case are you thinking?